hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WAL.java

```diff
@@ -193,10 +193,12 @@ default void sync(long txid, boolean forceSync) throws IOException {
   /**
    * Complete the cache flush.
    * @param encodedRegionName Encoded region name.
+   * @param maxFlushedSeqId The maxFlushedSeqId for this flush. There is no edit in memory that is
```

Comment:
maxFlushedSeqId should be just flushedSeqId? The 'max' confuses?  In fact, isn't this number the smallest sequenceid in the flush?

Reply:
Or dang, its fine. Ignore this comment. Its a nit and I had it wrong-way around thinking it the smallest edit flushed (wrong).